concrete/src/Page/ClonerOptions.php

```diff
@@ -160,27 +160,23 @@ public function setCopyAttributes($value)
     }
 
     /**
-     * Should the feature assignments be copied?
+     * Should the page type composer output block records be copied?
      *
      * @return bool
      */
-    public function copyFeatureAssignments()
+    public function copyPageTypeComposerOutputBlocks()
     {
-        return $this->copyFeatureAssignments;
+        return $this->copyPageTypeComposerOutputBlocks;
     }
 
     /**
-     * Should the feature assignments be copied?
+     * Should the page type composer output block records be copied?
      *
-     * @param bool $value
-     *
-     * @return $this
+     * @param bool $copyPageTypeComposerOutputBlocks
      */
-    public function setCopyFeatureAssignments($value)
+    public function setCopyPageTypeComposerOutputBlocks($copyPageTypeComposerOutputBlocks)
     {
-        $this->copyFeatureAssignments = (bool) $value;
-
-        return $this;
```

Comment:
I'd keep the `return $this` line: it's very handy for method chaining (and all the other setters already do it)